docs/content/customization/nutanix/machine-details.md

```diff
@@ -21,6 +21,9 @@ spec:
           controlPlane:
             nutanix:
               machineDetails:
+                additionalCategories: # This field is optional.
```

Comment:
@dlipovetsky pls refer to https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/commit/f17b1609a1faf1ee90b7cf2b1464c0b724a16239#diff-7be729f26dc4d92f63a4ca892c88ed87a9c4a4b018219c866e41d514c633a199R9. i would prefer that approach of required and then optional1..n as separate sections.

Reply:
Ahh I forgot to remove `additionalCategories` from the "required" section 🤦

Reply:
I'll do that in this PR :smile: